torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -477,6 +479,9 @@ struct CodeImpl {
   // map from unique of nodes to register in register table
   std::unordered_map<Value*, int> value_to_reg_;
 
+  // map from operator name to trailing unnecessary inputs
```

Comment:
Could you add an example in the comment? 
```
// Example: for a schema of aten::foo.str
// aten::foo.str(arg0: str="default", arg1: int=0, arg2: bool=False, arg3: float=0.0)
// If the usages in a graph is
// aten::foo("somestr", arg1=0, arg2=False, arg3=0.0)
// aten::foo("somestr", arg1=1, arg2=False, arg3=0.0)
// op_to_unnecessary_args_["aten::foo.str"] = ?
```

Reply:
sure